### ISSUE TITLE
ED Disco crash

### ISSUE THREAD

**Issue by Assimilator1** [EDDiscovery/EDDiscovery#2826]
### Description

 - No idea, just found it crashed when I quit ED

### Additional Information

 - see above

### Exception Details:

[QUOTE]EDDiscovery v11.4.5.0 There was an unhandled UI exception.

```

==== BEGIN ====

System.NullReferenceException: Object reference not set to an instance of an object.

   at EDDiscovery.UserControls.ScanDisplayUserControl.DrawSystem(SystemNode scannode, MaterialCommoditiesList curmats, HistoryList hl, String opttext, String[] filter) in C:\Code\EDDiscovery\EDDiscovery\UserControls\HelpersScanDisplayUC\ScanDisplayUserControl.cs:line 175

   at EDDiscovery.UserControls.UserControlScan.<DrawSystem>d__23.MoveNext() in C:\Code\EDDiscovery\EDDiscovery\UserControls\ScansStars\UserControlScan.cs:line 229

--- End of stack trace from previous location where exception was thrown ---

   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()

===== END =====

**Comment by robbyxp1** [comment]
Sounds like a random situation where the system got drawn as the control was being disposed of.

Not sure its worth covering.

**Comment by robbyxp1** [comment]
Its probable panelStars.

**Comment by Assimilator1** [comment]
Err, I've no idea what you meant by 'where the system got drawn as the control was being disposed of'.
lol.
What's panelstars?

Anyway, if you think it's a random one off & not worth worrying about, that's fine by me :).

**Comment by robbyxp1** [comment]
It was a note to myself.. Closing as i think its probably a one off.

**Comment by klightspeed** [comment]
I think this should be fixed in the next version (fixed in #2816 after you reported it in the forum thread).
